src/shabdawali.js

```diff
@@ -231,9 +231,24 @@ Shabdawali.prototype.typeText = function(cLine){
                 //this.element.textContent  += char;
                 this.element.textContent  = cLine.substr( 0, this.currentLetterIndex  );
                 var that = this;
-                setTimeout(function() {
+                if(char == ' ')
+                {
```

Comment:
Though I respect the style of every coder, I prefer to take starting parenthesis in the same line of loop, condition or other statement.

Reply:
Sure I'll update that.